include/translate-option.php

```diff
@@ -84,11 +98,10 @@ public function __construct( $name, $keys = array(), $args = array() ) {
 		add_action( 'update_option_' . $name, array( $this, 'update_option' ) );
 
 		// Sanitizes translated strings.
-		if ( empty( $args['sanitize_callback'] ) ) {
-			add_filter( 'pll_sanitize_string_translation', array( $this, 'sanitize_option' ), 10, 2 );
-		} else {
-			add_filter( 'pll_sanitize_string_translation', $args['sanitize_callback'], 10, 3 );
+		if ( ! empty( $args['sanitize_callback'] ) ) {
```

Comment:
Maybe it's out of scope, but it makes me think of a neat enhancement for this class.
We could pass the callback as a value for the keys, so each keys could have a different callback. But it might be hard to implement for array shaped callbacks...